### ISSUE TITLE
Expose Comment revision API

### ISSUE THREAD

**Issue by aslakknutsen** [openshiftio/openshift.io#1989]
Related to https://github.com/fabric8-services/fabric8-wit/issues/841

**Comment by nurali-techie** [comment]
I looked into current WIT API and here is what I understood for this requirement.

- Space can have more than one workitems.
- Workitem can have more than one comments.
- Each change in comment recorded as comment_revision.

Here is psql db data related to one comment and it's comment_revision.

`postgres=# select id, body, markup, created_at, updated_at from comments;
  id  |    body    |  markup  | created_at | updated_at           
------+------------+----------+------------+-------------
 1111 | comment_1a | Markdown | time1      | time2
(1 row)

postgres=# select id, comment_id, comment_body, comment_markup, revision_time, revision_type from comment_revisions ;
  id  | comment_id | comment_body | comment_markup | revision_time | revision_type 
------+------------+--------------+----------------+---------------+---------------
 2222 | 1111       | comment_1    | Markdown       | time1         |            1
 3333 | 1111       | comment_1a   | Markdown       | time2         |            4
(2 rows)
`
Note: GUID and timestamp replaced with readable values.

This issue is about exposing list API for comment_revisions table.

**API details:**
_Input:_ commentId
_Output:_ list of comment_revision for given id.  Each comment_revision should have id, comment_id, comment_body, comment_markup and other attributes.

In short, we will be adding list command for comment_revision which takes commentId as input and returns list of comment_revisions.

@aslakknutsen please review above requirement with API specification and provide your feedback.

**Comment by aslakknutsen** [comment]
@nurali-techie Yeah, this is a good start.
